File: 14.c
// average of N
#include <stdio.h>
int main(){
    int n,total=0,mean=0;
    
    printf("Enter number of elements:");
    if (scanf("%d", &n) != 1 || n <= 0) { // if the user enters a valid integer, scanf returns 1
        printf("Invalid input. Please enter a positive integer.\n");
        return 0;  
    } 
    int elements[n];

    for (int i = 0; i < n; i++){
        printf("Element %d:", i+1);
        scanf("%d", &elements[i]);
    }

    for (int i=0; i < n ; i++){
        total += elements[i];
    }
    mean = total/n;
    printf("%d",mean);

    return 0;


}

File: 14.py
# average of N

user_input = input("Enter elements separated by spaces: ")

elements = user_input.split()

try:
    elements = list(map(lambda x: float(x), elements))

    lenght = len(elements)
    total = sum(elements)
    mean = total/lenght
    print(f"Average is {mean}")

except ValueError:
    print("Enter numeric values only!")
